crates/oxc_language_server/src/main.rs

```diff
@@ -83,8 +83,8 @@ impl Options {
     }
 }
 
-#[tower_lsp::async_trait]
 impl LanguageServer for Backend {
+    #[expect(deprecated)] // TODO: FIXME
```

Comment:
Don't know what this is about 

```
warning: use of deprecated field `tower_lsp_server::lsp_types::InitializeParams::root_uri`: Use `workspace_folders` instead when possible
  --> crates/oxc_language_server/src/main.rs:89:19
   |
89 |         self.init(params.root_uri)?;
```

Reply:
LSP can support multiple workspaces, in VS Code we build the file URI with `workspaceFolders`:
https://github.com/oxc-project/oxc/blob/876b6ea9c033d4845a032327a88b322aa312431e/editors/vscode/client/extension.ts#L305-L309

We should fix this because we are telling the clients that we are supporting it:
https://github.com/oxc-project/oxc/blob/876b6ea9c033d4845a032327a88b322aa312431e/crates/oxc_language_server/src/capabilities.rs#L49-L52

Reply:
Ok, let's do this once this is merged.